portfolio/src/main/java/com/google/sps/servlets/ActivityDataServlet.java

```diff
@@ -12,20 +20,50 @@
 
 @WebServlet("/activity-data")
 public class ActivityDataServlet extends HttpServlet {
-  private Map<String, Integer> activityVotes = new HashMap<>();
+  private static final String ACTIVITY_OBJ = "Activity";
+  private static final String ACTIVITY = "activity";
+  private static final String ACTIVITY_COUNT = "count";
+  private DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
I think this can also be static and final?

Reply:
done